app/src/main/java/com/example/onsalestore/objects/PostItem.java

```diff
@@ -29,4 +31,6 @@ public PostItem(ClosetItem closetItem) {
 
     public void setUser() {ParseUser currentUser = ParseUser.getCurrentUser();put(KEY_USER, currentUser);}
 
+    public Integer getNumberOfLikes() {return getInt(KEY_LIKE);}
```

Comment:
This is not proper formatting.
```suggestion
    public Integer getNumberOfLikes() {
        return getInt(KEY_LIKE);
    }
```